### PR TITLE
c/clang:symbol dump demo

### PR DESCRIPTION
**Pull request by luoliwoshang**
- [x] filename get null

### CODE REVIEW

File: c/clang/_wrap/cursor.cpp [hunk 1]
```diff
@@ -1,29 +1,78 @@
...
 }
 
-extern "C" {
+extern "C"
+{
 
-CXString wrap_clang_getCursorSpelling(CXCursor *cur) {
-    return clang_getCursorSpelling(*cur);
-}
+    CXString wrap_clang_getCursorSpelling(CXCursor *cur)
```
Comment (xushiwei) [inline c/clang/_wrap/cursor.cpp]: use go style is better.

Comment (luoliwoshang) [reply c/clang/_wrap/cursor.cpp]: updated

File: c/clang/_wrap/cursor.cpp [hunk 2]
```diff
@@ -1,29 +1,77 @@
...
 }
 
-extern "C" {
+extern "C"
+{
 
-CXString wrap_clang_getCursorSpelling(CXCursor *cur) {
-    return clang_getCursorSpelling(*cur);
-}
+    CXString wrapCursorSpelling(CXCursor *cur)
```
Comment (xushiwei) [inline c/clang/_wrap/cursor.cpp]: should have clang namespace. I don't think `wrapCursorSpelling` is better than `wrap_clang_getCursorSpelling`.

bug I think:

```
CXString wrap_clang_getCursorSpelling(CXCursor *cur) {
}
```

is better than:

```
    CXString wrap_clang_getCursorSpelling(CXCursor *cur)
    {
    }
```

Ident spaces is not need for a global function.

Comment (luoliwoshang) [reply c/clang/_wrap/cursor.cpp]: updated, I have configured clang-format as follows, which basically aligns with some Go styles:
```
    "C_Cpp.clang_format_style": "{ BasedOnStyle: LLVM, IndentWidth: 4, ColumnLimit: 120, BreakBeforeBraces: Custom, BraceWrapping: { AfterFunction: false } }"
```

**Pull request merged**